akka-docs/src/main/paradox/stream/operators/Flow/asFlowWithContext.md

```diff
@@ -1,6 +1,6 @@
 # Flow.asFlowWithContext
 
-Turns a Flow into a FlowWithContext which can propagate a context per element along a stream.
+Turns a `Flow` into a `FlowWithContext` which can propagate a context per element along a stream.
```

Comment:
Not introduced in this PR, but maybe this description is misleading: this now suggests that you can take a `flow` that cannot propagate a context, and turn it into a `flow` that can propagate a context.

In reality, however, the `flow` should already allow somehow propagating the context, and `asFlowWithContext` just wraps the API so that that context is somewhat 'hidden' from the API.

Not sure how to convey that in a single sentence though ;)

Reply:
Good point, I gave it a try.